### DISCUSSION TITLE
how to bind a keymap 'jl' instead of 'esc' to enter 'normal-mod'

### DISCUSSION THREAD

**Discussion by 123dou** [ohmyzsh/ohmyzsh#10879] [Help]
i want to bind keymap 'jl' instead of 'esc' to enter 'normal-mod'. who can tell me,I'll very appreciate that.

**Comment by mcornella** [comment]
Put this in your zshrc file:
```sh
bindkey -M viins 'jl' vi-cmd-mode
```
Note that for multi-char keybindings to work, you need to press both faster than `$KEYTIMEOUT` milliseconds. You can change this variable to have more or less time between key presses.

**Reply by 123dou** [reply]
that's ok, thanks

**Reply by rwmitchell** [reply]
It’s better than just ok, it works!
